### ISSUE TITLE
Cannot find module '~/src/types' or its corresponding type declarations.

### ISSUE THREAD

**Issue by holwech** [scalar/scalar#3897]
### What happens?

We're experiencing issues when running `nuxi typecheck`:

```
node_modules/@scalar/nuxt/dist/runtime/components/ScalarApiReference.vue:9:36 - error TS2307: Cannot find module '~/src/types' or its corresponding type declarations.

9 import type { Configuration } from '~/src/types'
                                     ~~~~~~~~~~~~~


Found 1 error in node_modules/@scalar/nuxt/dist/runtime/components/ScalarApiReference.vue:9
```

Looking at `node_modules`, it seems like the `src` folder doesn't exist?

![image](https://github.com/user-attachments/assets/7abd44a6-e4bf-49cc-90e6-c58f12b73235)

This is with Scalar `0.2.141`. Tried to clean the build/cache folders and delete `node_modules` and build it again, but no luck so far.

### What did you expect to happen?

This is the same issue as #2944 which was closed a few months ago. I'm running Nuxt `3.13.1`.

**Comment by mirsella** [comment]
hello !

same with
@scalar/nuxt 0.2.160
nuxt 3.14.1592
node v23.4.0 or bun / pnpm

also happening in CI

thanks !

**Comment by holwech** [comment]
I think the Nuxt package is incorrectly built somehow. I don't remember exactly what the problem was unfortunately, since it's a while ago that I looked at it. In the end I wasn't able to fix the problem as I had problems building the Scalar project itself.

**Comment by holwech** [comment]
This bug breaks automated type checks in a CI pipeline, since this type error will always cause the checks to fail. Unfortunately, there is no way to ignore the error because it's located in an external dependency.

So while the bug might seem somewhat unimportant, the impact is pretty severe as it makes it impossible do automated type checking in a Nuxt project with Scalar installed.

**Comment by kong-dev** [comment]
Same issue here. Any updates? This issue broke the CI pipeline.

**Comment by hanspagel** [comment]
I don’t think this import (and thus the issue) exists anymore. Happy to reopen if someone has news on that. :)

**Comment by holwech** [comment]
@hanspagel still having the issue unfortunately and import seems to still be there. We're currently on `@scalar/nuxt": "^0.2.211`.

<img width="1178" height="817" alt="Image" src="https://github.com/user-attachments/assets/f21b1e88-c425-45f2-89fe-d2c38e9f0214" />
